### ISSUE TITLE
Arrays with both minItems and default misbehaves 

### ISSUE THREAD

**Issue by simonschmidt** [mozilla-services/react-jsonschema-form#631]
When providing both `default` (as a list)  and `minItems` to a field of type `array`
the defaults are treated as a single value that ends up in each field.

### Steps to Reproduce

[example](https://mozilla-services.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6e30sInNjaGVtYSI6eyJ0eXBlIjoib2JqZWN0IiwicHJvcGVydGllcyI6eyJsaXN0T2ZTdHJpbmdzIjp7InR5cGUiOiJhcnJheSIsInRpdGxlIjoiQSBmZXcgc3RyaW5ncywgd2VpcmQgd2hlbiBtaW5JdGVtcyBpcyBwcm92aWRlZCIsIml0ZW1zIjp7InR5cGUiOiJzdHJpbmcifSwiZGVmYXVsdCI6WyJoZWxsbyIsIndvcmxkIl0sIm1pbkl0ZW1zIjozfX19LCJ1aVNjaGVtYSI6e319)

```json
{
  "type": "object",
  "properties": {
    "listOfStrings": {
      "type": "array",
      "title": "A few strings, weird when minItems is provided",
      "items": {
        "type": "string"
      },
      "default": [
        "hello",
        "world"
      ],
      "minItems": 3
    }
  }
}
```
(try removing either `minItems` or `default` to see expected behavior)

#### Expected behavior
defaults should be treated as a list and be given one entry each, i.e.:

When `minItems > default.length` provide empty fields after the fields with the default values.
When `minItems <= default.length` provide one field per value in default list.

#### Actual behavior

The default items gets mashed together into each field

### Version

`0.49.0`

**Comment by olzraiti** [comment]
#579 would solve this.

**Comment by spacebaboon** [comment]
I think this can be closed now, with #663 merged.

**Comment by simonschmidt** [comment]
Yeap, thanks!
